src/test/coinstatsindex_tests.cpp

```diff
@@ -76,10 +76,12 @@ BOOST_FIXTURE_TEST_CASE(coinstatsindex_initial_sync, TestChain100Setup)
 
     BOOST_CHECK(block_index != new_block_index);
 
+    // Make sure coin_stats_index is not used by the scheduler thread after it
+    // has been destroyed.
+    SyncWithValidationInterfaceQueue();
+
     // Shutdown sequence (c.f. Shutdown() in init.cpp)
     coin_stats_index.Stop();
```

Comment:
If you move this, it would be good to update the other places as well.

My preference would be to use the same fix in all unit tests and not use a different fix for each unit test.

Reply:
That is just in `txindex_tests`, right? Moved.